### DISCUSSION TITLE
Do Code for IBM i notebooks use all markdown abilities?

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#676] [General]
I was trying to use the notebook feature (at 0.0.5) to build a local to my server to-do list for a large project.  When I attempt to insert task items they will not render as seen in other markdown sources. 

```- [ ]``` renders as a bullet and brackets. 

Is this a bug or a way notebooks render?

**Comment by worksofliam** [comment]
I am guessing this is the way Notebooks are.. but let me confirm also.

**Comment by m-tyler** [comment]
I installed `Markdown Preview Enhanced` and it renders an open markdown area properly. 
I also installed `REST Book` note  book extension and it renders tasks the same as does `Code for IBM i notebooks` 
Its just a problem with the notebook rendering. 

😕
